Command outfits & reprites

Davyei opened the pull request:
<!--
You are making this pull request for the Moffstation fork of Space Station 14.

Please be sure to follow general guidelines for upstream PRs, but also be sure to follow the Moffstation guidelines.
Guidelines: https://docs.spacestation14.io/en/getting-started/pr-guideline
See the Harmony contributing guidelines for an example on what we want: https://github.com/ss14-harmony/ss14-harmony/blob/master/CONTRIBUTING.md
-->

## About the PR
<!-- What did you change? -->
new command clothes for all command members & a new cloak for the pirate captain. _(the pirate command)_

also adds the items in this pr & previous prs into the uniform printer
## Why / Balance
<!-- Discuss how this would affect game balance or explain why it was changed. Link any relevant discussions or issues. -->
DRIP DRIP DRIP DRIP DRIP
Also Exbedio had command clothing for christmas time to be enabled for the season!
## Technical details
<!-- Summary of code changes for easier review. -->

## Media
<!-- Attach media if the PR makes in-game changes (clothing, items, features, etc).
Small fixes/refactors are exempt. Media may be used in SS14 progress reports with credit. -->
<img width="720" height="576" alt="Screenshot 2025-12-05 130642" src="https://github.com/user-attachments/assets/7643fd00-cdc2-42cf-991d-c27835803a67" />

## Requirements
<!-- Confirm the following by placing an X in the brackets [X]: -->
- [X] I have read and am following the [Upstream Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html) as well as the [Moffstation Contributing Guidelines](https://github.com/moff-station/moff-station-14/blob/master/CONTRIBUTING.md).
- [X] I have properly sectioned my changes into fork namespaces, and/or followed proper guidelines for modifying upstream files.
- [X] I have added media to this PR or it does not require an in-game showcase.
<!-- You should understand that not following the above may get your PR closed at maintainer’s discretion -->

## Breaking changes
<!-- List any breaking changes, including namespaces, public class/method/field changes, prototype renames; and provide instructions for fixing them.
This will be posted in #codebase-changes. -->

**Changelog**
<!-- Add a Changelog entry to make players aware of new features or changes that could affect gameplay.
Make sure to read the guidelines and take this Changelog template out of the comment block in order for it to show up.
Changelog must have a :cl: symbol, so the bot recognizes the changes and adds them to the game's changelog. -->
<!--
:cl:
- add: Added fun!
- remove: Removed fun!
- tweak: Changed fun!
- fix: Fixed fun!
-->
:cl: 
- add: new seasonal outfits & command clothing items
- tweak: Resprite existing captain & hop uniforms

Davyei committed:
Command Cloaks & Captain Outfit Items

Davyei committed:
cap's gas mask

Davyei committed:
some more outfits :3

Davyei committed:
whole lot more sprites

most are made by the wonderful exbedio on discord

Davyei committed:
more festive outfits

Davyei committed:
added clothing to the uniform printer

Davyei committed:
Loadouts!!

Davyei committed:
Moffstation designation

Davyei committed:
Update copyright information in meta.json

Signed-off-by: Dave <davidaperry2@gmail.com>

DuckManZach reviewed (approved):
Gave a smoke test ingame, seems to all be working

Davyei committed:
hat safety removed

Fried pointed out the hos' festive hat strap being too high on most species.

Centronias reviewed (changes requested):
Lots of missing guards in existing files, please rectify.

Davyei committed:
Missing Guards fixing

all the upstream files which were recoloured should now have the proper guarding

Centronias commented on Resources/Prototypes/Loadouts/loadout_groups.yml:
```diff
@@ -810,6 +824,7 @@
   minLimit: 0
   loadouts:
   - QuartermasterWintercoat
+  - QMFestivecoat
```
```suggestion
  - QMFestivecoat # Moffstation
```

Centronias reviewed (commented):
Trusting the linter for the RSIs and the loadout protos themselves.

Davyei committed:
Update Resources/Prototypes/Entities/Clothing/Neck/mantles.yml

Co-authored-by: Centronias <charlie.t.santos@gmail.com>
Signed-off-by: Dave <davidaperry2@gmail.com>

Davyei committed:
Update Resources/Prototypes/Loadouts/loadout_groups.yml

Co-authored-by: Centronias <charlie.t.santos@gmail.com>
Signed-off-by: Dave <davidaperry2@gmail.com>

Davyei committed:
Update Resources/Prototypes/Loadouts/loadout_groups.yml

Co-authored-by: Centronias <charlie.t.santos@gmail.com>
Signed-off-by: Dave <davidaperry2@gmail.com>

Davyei committed:
minor spelling mistake in .json copyright

Signed-off-by: Dave <davidaperry2@gmail.com>

Davyei committed:
Minor Spelling mistake in the .json

I couldn't spell Created apparently

Signed-off-by: Dave <davidaperry2@gmail.com>

Davyei committed:
Fix Spelling Mistake

Signed-off-by: Dave <davidaperry2@gmail.com>

Davyei committed:
Fix copyright wording in piratecap meta.json

Signed-off-by: Dave <davidaperry2@gmail.com>

Davyei committed:
Fix copyright text formatting in meta.json

Signed-off-by: Dave <davidaperry2@gmail.com>

Davyei committed:
Fix copyright wording in cmo.rsi meta.json

Signed-off-by: Dave <davidaperry2@gmail.com>

Davyei commented on Resources/Prototypes/_Moffstation/Recipes/Lathes/clothing.yml:
```diff
@@ -12,10 +12,15 @@
-  id: ClothingOuterCoatCaptainLongcoat
-  result: ClothingOuterCoatCaptainLongcoat
+  id: ClothingOuterWinterFestiveCaptain
+  result: ClothingOuterWinterFestiveCaptain
```
the existing long coat has the stats of the captain's carapace. thought having the ability to print carapaces may be a bit much
very much on purpose

Centronias reviewed (approved):
Running integration tests on this branch locally -- pretty sure the failure's fake, but I wanna make sure.
